index.html

```diff
@@ -300,5 +308,82 @@ <h3>Skills</h3>
       </div>
     </section>
     <!-- End of About-me section -->
+    <!-- Contact section -->
+    <section>
+      <div class="contact-container d-flex flex-dir-col">
+        <h3 class="contact-title">
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h3>
+        <form action="https://formspree.io/f/mayadege" method="post">
+          <ul class="d-flex flex-dir-col">
+            <li>
+              <label for="name"></label>
+              <input
+                type="text"
+                id="name"
+                name="user_name"
+                placeholder="Full name"
+              />
+            </li>
+            <li>
+              <label for="mail"></label>
+              <input
+                type="email"
+                id="mail"
+                name="user_email"
+                placeholder="Email adress"
+              />
```

Comment:
- [ ] Please follow the requirement for the contact-form Portfolio, and add `required` to the input fields.
![Capture d’écran 2021-07-13 233557](https://user-images.githubusercontent.com/61562626/125534646-41edc9fa-b234-4b30-94a7-76ed4f157a0d.png)